zap/src/main/java/org/zaproxy/zap/view/ContextTechnologyPanel.java

```diff
@@ -73,6 +73,8 @@ private TechnologyTreePanel getTechTree() {
             techPanel =
                     new TechnologyTreePanel(
                             Constant.messages.getString("context.technology.tree.root"));
+        } else {
+            techPanel.refresh();
```

Comment:
This prevents the techs from being saved.

Reply:
Ideas on fixing this? As stated before, awt.. is not my string suite.

Reply:
Reworked and working now.